pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.12"
+python = "^3.9"
```

Comment:
Can we confirm what ver Cloud is using? Also doesn't a lot of useful shit come into 3.10?

Reply:
`^3.9` means it allows for `3.9`, `3.10`, `3.11`, and `3.12`. But the cloud container image uses `3.11` so we should roll with that.

Reply:
yeah I see earlier in the README that we recommend 3.11, but let's enforce it so it's 1:1 with cloud

Reply:
Done